docs/deployment_guide/partner_editable/post_deployment.adoc

```diff
@@ -215,50 +215,50 @@
 |Parameter |Parameter Label |Default Value |Description
 
 |`AcceptEULAs`
-|Have all applicable VMware Tanzu Network EULAs been accepted?
+|Have all applicable {partner-registry} EULAs been accepted?
 |`No`
-|Choose `Yes` if you have already accepted the https://network.tanzu.vmware.com/products/tanzu-cluster-essentials/[Cluster Essentials for VMware Tanzu] and https://network.tanzu.vmware.com/products/tanzu-application-platform/[VMware Tanzu Application Platform] end-user license agreements (EULAs). TAP will fail to install successfully if both EULAs have not been accepted for the specified VMware Tanzu Network user account.
+|Choose `Yes` if you have already accepted the https://network.tanzu.vmware.com/products/tanzu-cluster-essentials/[Cluster Essentials for {partner-product-family-full}^] and https://network.tanzu.vmware.com/products/tanzu-application-platform/[{partner-product-name}^] end-user license agreements (EULAs). {partner-product-acronym} will fail to install successfully if both EULAs have not been accepted for the specified {partner-registry} user account.
 
 |`AcceptCEIP`
-|Have you read and accept the VMware CEIP policy?
+|Have you read and accept the {partner-company-short-name} CEIP policy?
 |`No`
-|Choose `Yes` if you have read and accepted the https://www.vmware.com/solutions/trustvmware/ceip.html[VMware customer experience improvement program (CEIP) policy].
+|Choose `Yes` if you have read and accepted the https://www.vmware.com/solutions/trustvmware/ceip.html[{partner-company-short-name} customer experience improvement program (CEIP) policy^].
```

Comment:
```suggestion
|Choose `Yes` if you have read and accepted the https://www.vmware.com/solutions/trustvmware/ceip.html[{partner-company-short-name} CEIP policy^].
```

"CEIP" is another acronym that appears to be defined at least 5 times in the guide. The first mention is in the `VMware Customer Experience Improvement Program (CEIP)` subheading title.